patch/src/lib.rs

```diff
@@ -189,7 +190,22 @@ fn list(
     repo: &git::Repository,
     profile: &Profile,
     project: &project::Metadata,
+    options: Options,
 ) -> anyhow::Result<()> {
+    // function uses information changed by sync
```

Comment:
I don't think the comment is necesssary, it confused me more than anything :stuck_out_tongue_closed_eyes: